src/main_chat/chatbot_manager.py

```diff
@@ -42,11 +43,11 @@ def ask_chatbot(self, user_input):  # this is key method which is called from ma
             print("U_Text = {}, u_conf = {}".format(u_text, u_conf))
             conf_res = u_conf > i_conf
             self._university_chatbot.inc_responses_in_row() if conf_res \
-                    else self._university_chatbot.reset_responses_in_row()
+                else self._university_chatbot.reset_responses_in_row()
             chatbot_response = u_text if conf_res else i_text
         else:
             self._university_chatbot.reset_responses_in_row()
             chatbot_response, c2 = self._ask_intro_chatbot(user_input)
             print('Intro Chatbot = ', user_input, ' c2 = ', c2)
-
+        self.db.add_new_doc_to_collection(configuration.RESPONSES_COLLECTION, response=chatbot_response)
         return chatbot_response
```

Comment:
please change the method content to be similar to the form of that method on branch master